tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -173,9 +174,6 @@ def test_it(
             },
             "mode": "basic-lti-launch",
             "rpcServer": {"allowedOrigins": ["http://localhost:5000"]},
-            "user": {
```

Comment:
@acelaya Moved this to the dashboard only config. sorry if this is a pain to sync with the FE.

Reply:
Not a pain. It's only used in an in-progress PR and it's easy to change.